docs/building-apps/governing-apps/managing/making-proposals.mdx

```diff
@@ -1013,4 +1159,4 @@ should include how to add and remove them.
 
 
 ### Register a new generic proposal
```

Comment:
@aterga , for the case study, I think the example would not work anymore as now one also has to define a topic for each proposal. Would it be easy to add a line to the example to fix this? (otherwise we can just remove the concrete example for now, but would of course be nicer to have it

Reply:
Removed the example for now as I think it is better to merge and add this back later when we had the time to review